src/Illuminate/Support/Number.php

```diff
@@ -342,6 +353,205 @@ public static function trim(int|float $number)
         return json_decode(json_encode($number));
     }
 
+    /**
+     * Return the absolute value of the given number.
+     *
+     * @param  int|float  $number
+     * @return int|float
+     */
+    public static function abs(int|float $number)
+    {
+        return abs($number);
+    }
+
+    /**
+     * Round the given number to the specified precision.
+     *
+     * @param  int|float  $number
+     * @param  int  $precision
+     * @param  int  $mode
```

Comment:
You can use a bit mask here as type:

```suggestion
     * @param  value-of<PHP_ROUND_*>  $mode
```